src/services/userServices.js

```diff
@@ -0,0 +1,21 @@
+const angular = require('angular');
+
+angular.module('app-bootstrap').factory('logIn', [
+  function login(email,password) {
```

Comment:
I believe you want the factory to return this function so you can use call from a controller